projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -135,10 +135,19 @@ export class CommentComponent implements OnChanges {
 
   formNgStyle() {
     return {
-      top: (this.rectTop * this.zoom) + 'px'
+      top: (this.rectTop * this.zoom) + 'px',
+      'z-index': this.selected ? 100 : 0
     };
   }
 
+  get commentBottomPos(): number {
+    return this.rectTop + this.height;
+  }
+
+  get height() {
+    return this.form.nativeElement.getBoundingClientRect().height / this.zoom;
```

Comment:
would the height change even if zoomed? i know top position will be effected but i dont see height changing (unless expanded/collapse)